README.md

```diff
@@ -40,22 +40,22 @@ sure that we're all on the same page.
   done the work _on theirs directly_. [Here][pr] is a great example of a pull
   request on the `Ruby` codebase.
 
-Let's go over a conceptual example. It's OK if this feels a bit confusing at
-first, you'll work through this countless times and eventually your brain and
-fingers will both grasp what's going on.
+Let's go over a conceptual, hypothetical example. It's OK if this feels a bit confusing at
+first. You'll work through this countless times and eventually your brain and
+fingers will both grasp what's going on. Let's look at the following example:
 
 ### Pull Request to a Source Repository
 
-1. Suppose you "fork" a repo from `https://github.com/learn-co-students/awesome-lab`.
-2. You now have a _copy_ of that repo on your Github account ("organization") i.e.
+1. Let's pretend we have a repo called `Awesome Lab` and we make a "fork" from this repo at `https://github.com/learn-co-students/awesome-lab`.
```

Comment:
instead of *we* let's be strict: "...pretend that the `learn-co-students` organization has a repo called `awesome-lab` and we make....

Reply:
Rest is good tho